DESCRIPTION

```diff
@@ -21,24 +21,23 @@ Imports:
   BiocGenerics (>= 0.16.1),
   cba (>= 0.2-15),
   cowplot (>= 0.9.3),
-  dplyr (>= 0.7.6),
+  data.table (>= 1.12.2),
   foreach (>= 1.4.3),
   GEOquery (>= 2.36.0),
   ggplot2 (>= 2.2.1),
   glmnet (>= 2.0-2),
   impute (>= 1.44.0),
+  iterators (>= 1.0.12),
   lazyeval (>= 0.2.1),
```

Comment:
I believe lazyeval was necessary for dplyr-related stuff. Check if it can be removed.

Reply:
Done with latest commit.